pkgs/top-level/aliases.nix

```diff
@@ -27,13 +27,6 @@ let
     then removeAttrs alias [ "recurseForDerivations" ]
     else alias;
 
-  # Disabling distribution prevents top-level aliases for non-recursed package
```

Comment:
I tried to [remove the `mapAttrs`](https://github.com/jtojnar/nixpkgs/commit/08daaaf2b3d118880165e9ca6336e46420b81322) to avoid having Nix forget the alias values containing `throw` come from the attribute set but it still does not remember:

```ShellSession
$ nix-build -A nixUnstable --show-trace
error:
       … while evaluating the attribute 'unstable'

         at /home/jtojnar/Projects/nixpkgs/pkgs/tools/package-management/nix/default.nix:220:5:

          219|   // {
          220|     unstable = throw "nixVersions.unstable has been removed. For bleeding edge (Nix master, roughly weekly updated) use nixVersions.git, otherwise use nixVersions.latest.";
             |     ^
          221|   }

       … while calling the 'throw' builtin

         at /home/jtojnar/Projects/nixpkgs/pkgs/tools/package-management/nix/default.nix:220:16:

          219|   // {
          220|     unstable = throw "nixVersions.unstable has been removed. For bleeding edge (Nix master, roughly weekly updated) use nixVersions.git, otherwise use nixVersions.latest.";
             |                ^
          221|   }

       error: nixVersions.unstable has been removed. For bleeding edge (Nix master, roughly weekly updated) use nixVersions.git, otherwise use nixVersions.latest.
```

So this will not help with https://discourse.nixos.org/t/rant-finding-errors-in-module-configs-after-incompatible-upgrades-is-dreadful/46375. But shorter stack traces are probably still better.